src/Components/Filtering/Filtering.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from 'react';
+import Button from '@mui/material/Button';
+// import Menu from '@mui/material/Menu';
+// import MenuItem from '@mui/material/MenuItem';
+import propTypes from 'prop-types';
+import { Search } from '@mui/icons-material';
+import { RangeFilter, gradeRangeMetadata, ageRangeMetadata } from './RangeFilter';
+import MultSelectElem from './MultiselectFilters';
+
+export default function Filter({ setRangeState, setMultiSelectInput }) {
+  const [tempRangeFilterData, setTempRangeFilterData] = useState({
+    age:
+    { min: ageRangeMetadata[0], max: ageRangeMetadata[18] },
+    grade: { min: gradeRangeMetadata[0], max: gradeRangeMetadata[13] },
+  });
+  const [tempMultiSelect, setTempMultiSelect] = useState({
+    'race/ethnicity': [],
+    identity_tags: [],
+    religion: [],
+    genre: [],
+    'theme/lessons': [],
+    book_type: [],
+  });
+
+  const handleSave = () => {
+    setRangeState(tempRangeFilterData);
+    setMultiSelectInput(tempMultiSelect);
+  };
+
+  const handleCancel = () => {
+    setTempRangeFilterData({
+      age:
+      { min: ageRangeMetadata[0], max: ageRangeMetadata[18] },
+      grade: { min: gradeRangeMetadata[0], max: gradeRangeMetadata[13] },
+    });
+    setTempMultiSelect({
+      'race/ethnicity': [],
+      identity_tags: [],
+      religion: [],
+      genre: [],
+      'theme/lessons': [],
+      book_type: [],
+    });
+    setRangeState({
+      age:
+      { min: ageRangeMetadata[0], max: ageRangeMetadata[18] },
+      grade: { min: gradeRangeMetadata[0], max: gradeRangeMetadata[13] },
+    });
+    setMultiSelectInput({
+      'race/ethnicity': [],
+      identity_tags: [],
+      religion: [],
+      genre: [],
+      'theme/lessons': [],
+      book_type: [],
+    });
+  };
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', rowGap: 10 }}>
+
+      <RangeFilter
+        rangeFilterData={tempRangeFilterData}
+        setRangeFilterData={setTempRangeFilterData}
+      />
+
+      <MultSelectElem
+        setTempMultiSelect={setTempMultiSelect}
+        tempMultiSelect={tempMultiSelect}
+      />
+
+      <div style={{ textAlign: 'right', textTransform: 'none', margin: '1vh 2vw 1vh 2vw' }}>
+        <Button onClick={handleCancel}>
+          Cancel
```

Comment:
Change the name of this button to Reset!
```suggestion
          Reset
```